packages/opentelemetry-exporter/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@instana/opentelemetry-exporter",
+  "version": "1.133.0",
+  "description": "OpenTelemetry Instana Exporter",
+  "keywords": [
+    "opentelemetry",
+    "exporter",
+    "nodejs",
+    "tracing"
+  ],
+  "author": "Willian Carvalho <willian.carvalho@instana.com>",
+  "homepage": "https://github.com/instana/nodejs/blob/main/packages/opentelemetry-exporter/README.md",
+  "license": "MIT",
+  "main": "src/index.js",
+  "directories": {
+    "lib": "src",
+    "test": "test"
+  },
+  "files": [
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/instana/nodejs.git"
+  },
+  "scripts": {
+    "start": "node test/app.js",
+    "debug": "node --inspect-brk test/app.js",
+    "test": "mocha --sort --reporter mocha-multi --reporter-options spec=-,xunit=../../test-results/opentelemetry-exporter/results.xml $(find test -iname '*test.js' -not -path '*node_modules*')"
+  },
+  "bugs": {
+    "url": "https://github.com/instana/nodejs/issues"
+  },
+  "dependencies": {
+    "@instana/serverless": "1.133.0",
+    "@opentelemetry/core": "0.25.0"
+  },
+  "devDependencies": {
+    "@instana/core": "1.133.0",
+    "eslint-plugin-mocha": "^9.0.0",
+    "mocha": "^7.2.0",
```

Comment:
There was a reason for having mocha and eslint in each package but I am not sure what the reason was :-) We can try to clean this up, but I'd it is out of scope for this PR.

Reply:
Good question.

My understanding of monorepo projects is that even though lerna manages dependencies in the root project, the sub project should still have all dependencies in its own package.json (the story is slightly different for package-lock.json.
This is because later on, when the package is published , lerna will no longer be present, so the package "is on its own".


But when it comes to dev dependencies, well, this may or may not be true. Most people will just use packages and I don't think many people would install dev dependencies or even run tests themselves, so having dev dependencies in the package.json may not make sense.

Does this make sense? Or am I missing something?
- Cheers